includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -145,6 +178,9 @@ public function update_settings( WP_REST_Request $request ) {
 		$this->update_account_statement_descriptor( $request );
 		$this->update_is_digital_wallets_enabled( $request );
 		$this->update_digital_wallets_enabled_locations( $request );
+		$this->update_digital_wallets_button_action_type( $request );
+		$this->update_digital_wallets_button_size( $request );
+		$this->update_digital_wallets_button_theme( $request );
```

Comment:
I think that would be a good improvement - is there a reason why these should be grouped and the existing ones shouldn't, tho? Do you think we could refactor everything separately?

Reply:
I think that would be a good improvement - is there a reason why these should be grouped and the existing ones shouldn't, tho? Do you think we could refactor everything separately? At this point I'm more inclined to have them separate, given the current design, but I'm open to suggestions 🤷

Reply:
> is there a reason why these should be grouped and the existing ones shouldn't, tho?

Nope! In regard to this one though, we can think of the digital wallets design settings as of a single entity ("the design") so I suggest just having one method taking care of all the fields, and maybe even a dedicated endpoint for modifying them since they live on a separate page?

As for the others, we could probably extract them to methods/classes on a by-card basis for updating, and keep all of them in the `GET` response because I think the only use case right now is loading all at once when loading the state.